dbtmetabase/parsers/dbt_manifest.py

```diff
@@ -28,7 +28,7 @@ def read_models(
         database = self.database
         schema = self.schema
         schema_excludes = self.schema_excludes
-        includes = self.includes
+        includes = [each_string.upper() for each_string in self.includes]
```

Comment:
You shouldn't modify the list, do it at the point of comparison.

Reply:
Thank you--I'd make the change right away, but I have a question about what that would look like.

I believe the point of comparison is `if (includes and model_name not in includes) or (model_name in excludes):`, which occurs twice in this file (lines 91 and 144).

Since `model_name` is always uppercase due to the transformation in line 49, wouldn't I have to transform the entire `includes` list to uppercase for `model_name not in includes` to evaluate properly?

I assume you don't want me to transform the list _twice_, so I'm not clear on what I should do differently, or why it's a problem to modify the list earlier on to avoid repetition since it is used later in the file twice.

I guess I could iterate through the list and transform each element as I go, but why write more code?

Not trying to be difficult here--I appreciate any clarification you can provide.

Reply:
Ahhh, I see your point. In that case, yes, transform the whole list for includes and excludes. I would prefer if you lowercased them instead of uppercase (that's the preferred format in dbt).

Have a look at the CI error as well, I think the includes/excludes can be None so it warns that your list comprehension could crash.

Reply:
Thank you! I think I've addressed everything you brought up. 

I also made one edit to `dbtmetabase/models/metabase.py` to accommodate special_type in MetabaseColumn. I think this was an accidental omission since special type is included in METABASE_META_FIELDS defined in line 9 of the same file. It would be awesome if that update could be included in this PR but I understand if not since it's unrelated.

Reply:
Not accidental, Metabase renamed `special_type` to `semantic_type`. Please revert that change.

Reply:
Ah, thank you for letting me know. I just reverted the change.